<!-- WRS 2018 Handyman SLAM.launch -->
<launch>
  <!-- SLAM -->
  <!--include file="$(find hsr_ros)/launch/slam.launch"-->
  <include file="$(find sobit_navigation)/launch/slam_wrs_hsr.launch">
    <arg name="map_file" default="$(find robocup_simopl_handyman)/map/test/map.yaml"/>
    <arg name="map_movebase_file" default="$(find robocup_simopl_handyman)/map/test/map_movebase.yaml"/>
    <arg name="location_file" default="$(find robocup_simopl_handyman)/map/test/Environment_01_grasping.json"/>
    <arg name="look_down_floor_flag" default="false"/>
    <arg name="look_at_goal_way_flag" default="false"/>
  </include>

  <group ns="handyman_tf_broadcaster">
    <param name="map_frame_name" type="string" value="map"/>
    <param name="other_obj_json_path" type="string" value="$(find robocup_simopl_handyman)/map/test/Environment_01_destination.json"/>
    <node name="other_obj_broadcaster_node" pkg="robocup_simopl_handyman" type="other_obj_broadcaster"/>
  </group>
</launch>
